deployment/dev/.env

```diff
@@ -0,0 +1,2 @@
+# Docker by default uses the folder name to isolate different projects. This might however be a problem because we call our folder "dev", which might also be how other projects call their folders. To avoid any collisions, we use an idea from https://stackoverflow.com/questions/50947938/docker-compose-orphan-containers-warning
```

Comment:
Please shorten to "namespacing to avoid container name collision"

Reply:
Done 👍